Perl.gitignore

```diff
@@ -18,3 +18,5 @@ nytprof.out
 *.o
 *.bs
 /_eumm/
+/*.zip
```

Comment:
Could you include a comment before this line to indicate which tools are responsible for this?

I know this hasn't really been done before in this file, but it can't hurt to start annotating things...